### PR TITLE
Wave function

### PR DESCRIPTION
**Pull request by i-be-keggles**
Fixed some formatting errors (there were annoying commas everywhere and that gap space at the top). Also added gradient falloff. Spent like an hour fixing some tip behind thing. I love git.

### Summary by CodeRabbit

Release Notes:

- New Feature: Added a GitHub Actions workflow for syncing branches between two repositories.
- Documentation: Updated the main README with contribution guidelines, workflow visual guide, and resources. Also updated the Public Frontend README with setup instructions and tech stack details.
- Refactor: Improved the wave generation logic in `MatrixBackground.astro` and organized the code into a module.
- New Feature: Introduced a GitHub rankings table in `Scoreboard.astro`, displaying user contributions with a progress bar.
- New Feature: Implemented a dynamic word cloud in `WordCloud.astro`, showcasing an array of words in random positions.
- Chore: Integrated Matomo tracking code for analytics in `Blog.astro` and `Layout.astro`.
- Style: Incorporated the "Iosevka" font across the application in `Layout.astro`.
- Refactor: Reorganized the layout structure in `index.astro` and introduced new components.

### CODE REVIEW

**Comment by velocitatem** [comment]
💀

**Comment by velocitatem** [comment]
I think there may have been some rebase error on your fork, we can take a look later today.

File: fe-public/src/components/Scoreboard.astro [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+import getSortedPosts from "../scripts/load-posts";
+
+let averageCommitContent = 0;
+
+let users = [
```
Comment (vtwoptwo) [inline fe-public/src/components/Scoreboard.astro:6]: users should be fetched into a variable / not hard coded.

Comment (vtwoptwo) [reply fe-public/src/components/Scoreboard.astro:6]: Im assuming this will be resolved with resolving the MR conflict

Comment (velocitatem) [reply fe-public/src/components/Scoreboard.astro:6]: They are, it's just the old code that somehow made it into this PR.

**Comment by i-be-keggles** [comment]
Git why. I changed 1 file. Gonna reset the branch.
Amazing that this was your first review @vtwoptwo. The rest of our code is fine I swear.

**Pull request closed without merging**